src/components/Toast.tsx

```diff
@@ -71,6 +87,33 @@ class Toast extends React.PureComponent<ToastProps, ToastState> {
   render() {
     const bColor = _colors[this.state.type];
 
+    const customStyles = {
```

Comment:
we can move this outside render so new objects aren't created on each re-render. It this case it won't matter a lot but it will be good if we follow similar approach everywhere.

Reply:
done